azkaban-common/src/main/java/azkaban/project/JdbcProjectImpl.java

```diff
@@ -95,6 +98,9 @@ public JdbcProjectImpl(final Props props, final DatabaseOperator databaseOperato
         logger.info("project temporary folder already existed.");
       }
     }
+    int maxFlowFileSizeInMB = props.getInt("max.flow.file.size.mb", MAX_FLOW_FILE_SIZE_IN_MB_DEFAULT);
+    maxFlowFileSizeInBytes = maxFlowFileSizeInMB * 1024 * 1024;
```

Comment:
I think you might use the `MAX_FLOW_FILE_SIZE_KEY` variable.

Also, this value will be used by web-server and be honored as long as it's running?

Reply:
Thank you for spotting that @tsangz2013. Fixed and updated the PR. Yes, I did want to use the const over there. This property is like most of the properties we have. Loaded when web server starts, and the value is used throughout the lifetime of the process. If we want to change, a web server restart is required.